From 665a97574cc627459e754bc42ce5fbf76a02d1a0 Mon Sep 17 00:00:00 2001
From: SeongJae Park <sj@kernel.org>
Date: Thu, 27 Jun 2024 11:16:45 -0700
Subject: [PATCH] Docs/mm/index: Remove 'Memory Management Guide' chapter
 marker

'Memory Management Guide' chapter aims to be not an additional chapter
of the document, but the ultimate outline that additonal 'legacy'
documents that still need will eventually be integrated into.  In the
sense, marking it as a chapter under the document makes no sense.
Remove the chapter marker.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/index.rst | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Documentation/mm/index.rst b/Documentation/mm/index.rst
index 48b9b559ca7b..5007815405f1 100644
--- a/Documentation/mm/index.rst
+++ b/Documentation/mm/index.rst
@@ -2,9 +2,6 @@
 Memory Management Documentation
 ===============================
 
-Memory Management Guide
-=======================
-
 This is a guide to understanding the memory management subsystem
 of Linux.  If you are looking for advice on simply allocating memory,
 see the :ref:`memory_allocation`.  For controlling and tuning guides,
-- 
2.39.2

